server/src/main/java/io/druid/server/ClientInfoResource.java

```diff
@@ -300,6 +303,24 @@ public int compare(Interval o1, Interval o2)
     return metrics;
   }
 
+  @GET
+  @Path("/{dataSourceName}/candidates/intervals/{intervals}/numCandidates/{numCandidates}")
```

Comment:
i guess after the changed in https://github.com/druid-io/druid/pull/2424 by @pjain1 we need to add Auth logic ?